bin/send_email.py

```diff
@@ -143,31 +145,41 @@ def send_email(view_id: str, submission_id: str, email_with_score: str):
     # Get the Synapse users to send an e-mail to
     ids_to_notify = helpers.get_participant_id(syn, submission_id)
 
-    # Create the subject and body of the e-mail message, depending on submission status
-    subject = (
-        f"Evaluation Success: {submission_id}"
-        if submission_annotations.status == "VALIDATED"
-        else f"Evaluation Failed: {submission_id}"
-    )
-    body = email_template(
-        submission_annotations.status,
-        email_with_score,
-        submission_id,
-        view_id,
-        submission_annotations.score,
-        submission_annotations.reason,
-    )
+    # Create the subject and body of the e-mail message, depending on
+    # the notification type and submission status:
+    if notification_type.upper() == "BEFORE":
+        # Before-evaluation notification
+        subject = f"Evaluation Started: {submission_id}"
+        body = (
+            f"Dear participant,\n\n"
+            f"Your submission <b>{submission_id}</b> is now being evaluated. "
+            "We will notify you again once the evaluation completes.\n\n"
+            "Thank you for your participation!\n\n"
+            "The Challenge Organizers"
```

Comment:
Is there any way or need to customize the messages we send a little more so the participant can see a direct connection to the specific challenge they are participating in? Also maybe we address them by their Synapse username?

Reply:
Good point! Users might be participating in multiple challenges at once. I'll add the challenge name and the participant username in the template.